sbt-plugin/src/sbt-test/legacy/legacy/build.sbt

```diff
@@ -14,7 +14,7 @@ version := scalaJSVersion
 
 val versionSettings = Seq(
   version := scalaJSVersion,
-  scalaVersion := "2.11.12"
+  scalaVersion := "2.12.8",
```

Comment:
This change actually concerns me. The sbt plugin test was the only thing in our CI that would ensure that Scala.js works at all when using a `scalaVersion` that is different from the sbt plugin's version of Scala. Using 2.11.12 was quite intentional here. Switching to 2.12.8 digs a large hole in our test coverage.

I understand the difficulty of doing `publishLocal` of two sets of artifacts with differing `scalaVersion`s when using `scripted`, but I think we shouldn't lose the above coverage.

Reply:
Ah... Is this even possible with scripted?

Reply:
I don't know. I'll ask in the gitter channel.